### DISCUSSION TITLE
Add Haystack integration

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#1221] [Ideas]
### Describe the feature or potential improvement

Callback handler for haystack
https://github.com/deepset-ai/haystack

Chainlit integration with haystack might be a good reference implementation
https://docs.chainlit.io/integrations/haystack

**Comment by sandangel** [comment]
This will be awesome.

**Comment by marcklingen** [comment]
Forgot to update this thread, this integration is available for some time already: https://langfuse.com/docs/integrations/haystack/get-started
